### PR TITLE
Update Cinnamon to version 2.15.0

### PR DESCRIPTION
**Pull request by marcospereira**
- Update Cinnamon version to 2.15.0
- Add missing Akka HTTP modules
- Remove duplicated akka-coordination

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -61,12 +60,14 @@ object Dependencies {
 
   val akkaHttp = Seq(
     "com.typesafe.akka" %% "akka-http"            % AkkaHttp,
+    "com.typesafe.akka" %% "akka-http-caching"    % AkkaHttp,
```
Comment (marcospereira) [inline project/Dependencies.scala:63]: I'm pretty sure the list of modules [here](https://developer.lightbend.com/docs/introduction/getting-help/build-dependencies.html#_akka_http) is incomplete.

Comment (ignasi35) [reply project/Dependencies.scala:63]: That list is incomplete (and not just for Akka HTTP). Just noticed there are other Lagom artifacts missing too.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -32,7 +32,6 @@ object Dependencies {
     "com.typesafe.akka" %% "akka-actor"                  % Akka,
     "com.typesafe.akka" %% "akka-actor-testkit-typed"    % Akka,
     "com.typesafe.akka" %% "akka-actor-typed"            % Akka,
-    "com.typesafe.akka" %% "akka-coordination"           % Akka,
```
Comment (ignasi35) [inline project/Dependencies.scala:35]: HawkEye Pereira!

**Review by ignasi35 — APPROVED** [review]

File: project/Dependencies.scala (hunk 1 above)
Comment (ignasi35) [reply project/Dependencies.scala:63]: I've raised https://github.com/lightbend/lightbend-technology-intro-doc/issues/240

**Pull request merged**